ui/src/components/common/Header.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import {
+  Dropdown,
+  DropdownToggle,
+  DropdownItem,
+  Page,
+  PageHeader,
+  Toolbar,
+  ToolbarGroup,
+  ToolbarItem
+} from '@patternfly/react-core';
+import { UserIcon } from '@patternfly/react-icons';
+import accessibleStyles from '@patternfly/patternfly/utilities/Accessibility/accessibility.css';
+import { css } from '@patternfly/react-styles';
+import './Header.css';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { toggleHeaderDropdown } from '../../actions/actions-ui';
+
+class Header extends React.Component {
+  onTitleClick = () => {};
+
+  onUserDropdownToggle = () => {
+    const isUserDropdownOpen = this.props.isUserDropdownOpen;
+    this.props.toggleHeaderDropdown(isUserDropdownOpen);
+  };
+
+  onUserDropdownSelect = () => {
+    this.props.toggleHeaderDropdown();
+  };
+
+  onLogoutUser = () => {
+    console.log('onLogoutUser()');
+  };
+
+  render () {
+    const logoProps = {
+      onClick: () => this.onTitleClick()
+    };
+
+    const toolbar = (
+      <Toolbar>
+        <ToolbarGroup className={css(accessibleStyles.screenReader, accessibleStyles.visibleOnLg)}>
+          <UserIcon />
+          <ToolbarItem className={css(accessibleStyles.screenReader, accessibleStyles.visibleOnMd)}>
+            <Dropdown
+              isPlain
+              position="right"
+              onSelect={this.onUserDropdownSelect}
+              isOpen={this.props.isUserDropdownOpen}
+              toggle={<DropdownToggle onToggle={this.onUserDropdownToggle}>currentUserName</DropdownToggle>}
```

Comment:
@damienomurchu we'll probably do the implemenation later, but could you move the `currentUserName` hardcoded value to Redux state?